src/sentry/api/endpoints/organization_events_stats.py

```diff
@@ -74,3 +89,29 @@ def get_field(self, request, snuba_args):
         snuba_args["aggregations"] = [aggregate]
 
         return snuba_args
+
+
+def cache_key_for_snuba_args(organization, params, snuba_args):
+
+    lol = json.dumps(
+        [
+            snuba_args.get("aggregations"),
+            snuba_args.get("conditions"),
+            snuba_args.get("filter_keys"),
+            params,
+        ]
+    )
+
+    import logging
+
+    logging.info("payload: %s", lol)
+
+    hash_result = hash(lol)
```

Comment:
You might want to use `hashlib.md5()` here as hash() may not give you enough variance for cache keys.

Reply:
Using `sha1` for this.